WPL/AI-usage.md

```diff
@@ -0,0 +1,693 @@
+   <td>2</td>
+   <td>0</td>
+   <td>0</td>
+   <td>0</td>
+   <td>0</td>
+   <td>0</td>
+   <td>0</td>
+   <td>0</td>
+   <td>0</td>
+   <td>2</td>
+  </tr>
+  <tr>
+   <td>30/4/2024</td>
+   <td>0</td>
+   <td>0</td>
+   <td>0</td>
+   <td>0</td>
+   <td>0</td>
+   <td>0</td>
+   <td>0</td>
+   <td>0</td>
+   <td>1</td>
+   <td>1</td>
+  </tr>
+  <tr>
+   <td>07/5/2024</td>
+   <td>1</td>
+   <td>0</td>
+   <td>0</td>
+   <td>0</td>
+   <td>0</td>
+   <td>0</td>
+   <td>0</td>
+   <td>0</td>
+   <td>2</td>
+   <td>3</td>
+  </tr>
+  <tr>
+   <td>Total</td>
+   <td>32</td>
+   <td>1</td>
+   <td>1</td>
+   <td>2</td>
+   <td>1</td>
+   <td>1</td>
+   <td>1</td>
+   <td>1</td>
+   <td>3</td>
+   <td>43</td>
+  </tr>
+</table>
+
+
+
+La primera semana que se empezó a registrar los usos de las inteligencias artificiales sólo teníamos tareas de redacción de documentos, por lo que todas las consultas fueron a modelos de aprendizaje de lenguaje (Chat Gpt). Además había un gran volumen de documentos con los que no teníamos experiencia previa, por lo que hubo que preguntar sobre el contenido o la estructura de ellos. 
+
+La siguiente semana comenzó el desarrollo, por lo que las consultas son de mayor relación a la programación. De los 4 usos que se le han dado a las inteligencias artificiales, un 75% han sido respecto a comienzo de sistemas informáticos. 
+
+La semana del 25/2/2024 se hacen dos consultas a Chat GPT. También apuntamos que 5 miembros usan _Github Copilot_, que no requiere prompts, si no ofrece posible código a escribir en función del contenido actual del fichero.
+
+La semana del 3/3/2024 se hacen 3 consultas a Chat Gpt, 2 de ellas respecto a programación y otra para definir el nuevo método de evaluación individual.
+
+La semana del 10/3/2024 se hacen 5 consultas a Chat Gpt. 2 de ellas son respecto al documento del impacto legal, que es un tema muy delicado y es pertinente preguntar a inteligencias artificiales para tener una dirección general. Cabe recalcar una petición sobre errores en JSON, que es un problema que ha tenido que afrontar el frontend debido a la complejidad de mandar documentos mediante JSON.
+
+La semana del 26/3/2024 se hacen 3 consultas a Chat Gpt. 2 de ellas son respecto al documento del impacto legal y sobre el commitment agreement de los usuarios pilotos. La tercera fue respecto a como realizar un formulario para una peticion con unos determinados atributos y 1 uso de Copilot. Además, el anuncio se ha realizado con la ayuda de SunoAI.
+
+La semana 1 del Sprint 3, es decir la semana de 03/04/2024 se hacen 5 consultas a Chat Gpt. De ellas, 2 no se lograron resolver haciendo uso de la ia, lo que provocó que se perdiera en total 1:35 horas. Por otro lado, el resto de consultas provocarón un ahorro de horas significativo, con un total de 6:40 horas. De las consultas, 1/7 fueron realizadas para tareas de documentación. En la siguiente gráfica se muestra el nivel de satisfación frente al ahorro del tiempo estimado:
```

Comment:
I would recommend capitalizing "ia" to "IA" due to it being initials and to keep a constant wording all along the document.